packages/selleo-design-core/src/Button.tsx

```diff
@@ -1,9 +1,25 @@
 import { h } from "preact";
 
-export function Button() {
-  return <button class="inline-flex bg-orange-500">Button</button>;
+export function PrimaryButtonSmall() {
+  return <button class="inline-flex justify-center text-sm text-white font-extrabold leading-7 tracking-normal min-w-[124px] min-h-[32px] bg-brand-primary hover:bg-brand-primary-600 focus:bg-brand-primary-600 focus:outline focus:outline-offset-1 focus:outline-1 focus:outline-brand-primary-600 rounded disabled:bg-brand-primary-500 disabled:opacity-30">Text</button>;
+}
+
+export function PrimaryButton() {
+  return <button class="inline-flex justify-center text-base text-white font-extrabold leading-7 tracking-normal min-w-[160px] min-h-[48px] bg-brand-primary hover:bg-brand-primary-600 focus:bg-brand-primary-600 focus:outline focus:outline-1 focus:outline-brand-primary-600 focus:outline-offset-2 rounded disabled:bg-brand-primary-500 disabled:opacity-30">Text</button>;
+}
+
+export function PrimaryOutlinedButtonSmall() {
+  return <button class="inline-flex justify-center text-sm font-extrabold min-w-[124px] min-h-[32px] text-brand-primary border border-solid border-brand-primary bg-white hover:bg-brand-primary-100 focus:bg-brand-primary-100 focus:outline focus:outline-1 focus:outline-offset-2 focus:outline-brand-primary rounded disabled:bg-white disabled:opacity-30">Text</button>;
 }
 
-export function ButtonExample() {
-  return <button class="inline-flex bg-orange-700">Button</button>;
+export function PrimaryOutlinedButton() {
```

Comment:
I guess the Background should be clear instead of white @michalgren am i right?

Reply:
IMO It makes more sense like this indeed, for now changed according to @k1eu comments 👍